### DISCUSSION TITLE
Jest encountered an unexpected token

### DISCUSSION THREAD

**Discussion by Alex18595** [chakra-ui/panda#2288] [General]
### Description

When running a test that contains imports from pandacss, an error occurs - "Jest encountered an unexpected token". Details:
 /home/alex/alxProject/base2/styled-system/css/index.mjs:1 ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){**export** * from './css.mjs';

### Link to Reproduction

https://stackblitz.com/edit/vitejs-vite-xjh7xf?file=src%2FApp.test.tsx

### Steps to reproduce

2. Click on '...'
3. Scroll down to '...'

### JS Framework

React, typescript.

### Panda CSS Version

0.32.1

### Operating System

- [ ] macOS
- [X] Windows
- [ ] Linux

**Comment by Alex18595** [comment]
Is there any solution to this problem?

**Reply by astahmer** [reply]
yes @Alex18595 
add `"allowJs": true,` to your `tsconfig.json`
add `'^.+\\.(ts|tsx|js|jsx)?$': 'ts-jest',` to your `transform` in `jest.config.ts` 

https://stackblitz.com/edit/vitejs-vite-m1btiz?file=jest.config.ts,tsconfig.json

**Reply by Alex18595** [reply]
The problem remains, the same error
![test](https://github.com/chakra-ui/panda/assets/89704634/ab303619-e03b-47a8-a8f0-f08f5e289d2d)

**Reply by astahmer** [reply]
ah yes sorry I forgot to mention you also need `outExtension: 'js',` in `panda.config.ts`

**Reply by Alex18595** [reply]
Thank you. Everything is working.
